dspace-api/src/test/java/org/dspace/statistics/export/processor/ExportEventProcessorTest.java

```diff
@@ -0,0 +1,329 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.statistics.export.processor;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.UnsupportedEncodingException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.http.HttpServletRequest;
+
+import org.dspace.AbstractDSpaceTest;
+import org.dspace.content.Entity;
+import org.dspace.content.EntityType;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.EntityService;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * Test for the ExportEventProcessor class
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ExportEventProcessorTest extends AbstractDSpaceTest {
+
+    @Mock
+    private Context context = mock(Context.class);
+    @Mock
+    private HttpServletRequest request = mock(HttpServletRequest.class);
+    @Mock
+    private Item item = mock(Item.class);
+
+    private ConfigurationService configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+    @Mock
+    private ItemService itemService = mock(ItemService.class);
+    @Mock
+    private EntityService entityService = mock(EntityService.class);
+
+
+    @InjectMocks
+    ExportEventProcessor exportEventProcessor = mock(ExportEventProcessor.class);
+
+    @Before
+    public void setUp() {
+        configurationService.setProperty("stats.tracker.enabled", true);
+    }
+
+    @Test
+    /**
+     * Test the getBaseParameters method
+     */
+    public void testGetBaseParameters() throws UnsupportedEncodingException {
+        exportEventProcessor.context = context;
+        exportEventProcessor.request = request;
+        exportEventProcessor.configurationService = configurationService;
+        exportEventProcessor.trackerUrlVersion = "Z39.88-2004";
```

Comment:
This should be set via `configurationService.setProperty("stats.tracker.urlversion", "Z39.88-2004")` instead of modifying the class directly.  That'd make this test better as it ensures the configuration is working properly.